kai/reactive_codeplanner/task_runner/compiler/compiler_task_runner.py

```diff
@@ -146,6 +146,11 @@ def execute_task(self, rcm: RepoContextManager, task: Task) -> TaskResult:
     def parse_llm_response(self, message: BaseMessage) -> MavenCompilerLLMResponse:
         """Private method that will be used to parse the contents and get the results"""
 
+        if isinstance(message.content, list):
+            return MavenCompilerLLMResponse(
+                reasoning="", java_file="", additional_information=""
```

Comment:
don't quite follow this

Reply:
basically, we don't know how to parse llm responses that are not just a string.

we need to handle this in the future